doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -498,6 +498,50 @@ in :ref:`canonicalstructures`; here only a simple example is given.
    Otherwise said, :token:`qualid` is canonically used to extend the field |c_i|
    into a complete structure built on |c_i|.
 
+   The following kinds of terms are supported for the fields |c_i| of :token:`qualid`:
```

Comment:
Do not introduce new `|c_i|`. This is a terrible practice that I'm trying to get rid of (and there aren't much instances remaining). FTR, this relies on these replacement rules: https://github.com/coq/coq/blame/master/doc/sphinx/refman-preamble.rst

Reply:
I don't understand what you mean by introducing new `|c_i|`. It is already used in the paragraphs above, isn't it? What exactly does it mean to (newly) introduce it? I haven't worked with sphinx so maybe I am missing something obvious here.

Reply:
Sorry, what I mean is that this `|c_i|` is not a standard way of writing mathematical expressions in Sphinx, it is an abuse of the replacement rules system. If you want to write mathematical expressions, you may use the `:math:` environment to put some LaTeX, but if all you want is to have indices, then there are better ways than this. In particular, `:n:` supports inserting indices with the following syntax: ``` :n:`c__i` ```.

>It is already used in the paragraphs above, isn't it?

I'd like to fix this.

Reply:
Okay, got it! I'll use `:n:`.

> I'd like to fix this.

Should this happen in another PR or in this one here?

Reply:
I decided to just include that change.

Reply:
It could be in this one, but I suggest waiting after #12239 is merged, otherwise rebasing risks becoming more difficult.

Reply:
Oh, a race condition. :) I think it should be fine as long as I don't squash my commits too early.